dictcc.el

```diff
@@ -203,8 +213,7 @@ At the moment they are of the form `<tr id='trXXX'></tr>'."
 
 (defun dictcc--candidate (pair)
   "Generate the candidate pair for a PAIR of translations."
-  (let* ((format-string (format "%%-%ds -- %%%ds"
-                                dictcc-candidate-width
+  (let* ((format-string (format "%%-%ds   %%s"
```

Comment:
Could you just briefly explain why you changed this? Is it only so that the translation is unconstrained in length?